radio/src/lua/interface.cpp

```diff
@@ -954,6 +954,27 @@ bool luaTask(event_t evt, uint8_t scriptType, bool allowLcdUsage)
   return scriptWasRun;
 }
 
+void checkLuaMemoryUsage()
+{
+#if (LUA_MEM_MAX > 0)
+  uint32_t totalMemUsed = luaGetMemUsed(lsScripts);
+#if defined(COLORLCD)
+  totalMemUsed += luaGetMemUsed(lsWidgets);
+  totalMemUsed += luaExtraMemoryUsage;
+#endif
+  if (totalMemUsed > LUA_MEM_MAX) {
```

Comment:
Did not come up with smarter solution, so in this case both Lua states are disabled.